eraser-map.yaml

```diff
@@ -902,6 +902,13 @@ layers:
                 priority: 1
                 sprite: capital-xl
                 size: 18px
+    route_icon:
+        data: { source: route_icon }
+        draw:
+            icons:
+                priority: 1
+                sprite: airport
+                size: 18px
```

Comment:
Nice!

On Fri, Oct 30, 2015 at 9:20 AM, Karim Naaji <notifications@github.com>
wrote:

> In eraser-map.yaml
> <https://github.com/tangrams/eraser-map/pull/67#discussion_r43521452>:
>
> > @@ -902,6 +902,13 @@ layers:
> >                  priority: 1
> >                  sprite: capital-xl
> >                  size: 18px
> > +    route_icon:
> > +        data: { source: route_icon }
> > +        draw:
> > +            icons:
> > +                priority: 1
> > +                sprite: airport
> > +                size: 18px
>
> Also it would make sense to setup transition to not fade:
>
> route_icon:
>     transition:
>         [show, hide]:
>             time: 0s
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/tangrams/eraser-map/pull/67/files#r43521452>.
>